installing/installing_ibm_z/installing-ibm-z.adoc

```diff
@@ -44,7 +44,9 @@ include::modules/csr-management.adoc[leveloffset=+2]
 
 * See link:http://public.dhe.ibm.com/software/dw/linux390/perf/zvm_hpav00.pdf[Scaling HyperPAV alias devices on Linux guests on z/VM] for performance optimization.
 
-* See xref:../../scalability_and_performance/ibm-z-recommended-host-practices.adoc#ibm-z-recommended-host-practices[Recommended host practices for IBM Z & LinuxONE environments].
+* link:https://www.vm.ibm.com/library/presentations/lparperf.pdf[Topics in LPAR performance]
```

Comment:
I can't get this link to load. Please double-check that you don't need to log in somewhere else to view it. I also suggest adding a description of the kind of external link that this is, similar to "in IBM Documentation" in a previous link.

Reply:
@kalexand-rh I think vm.ibm.com was down yesterday it didn't work for me either now it does.